addons/docs/src/frameworks/web-components/__testfixtures__/lit-element-demo-card/properties.snapshot

```diff
@@ -2,24 +2,58 @@
 
 exports[`web-components component properties lit-element-demo-card 1`] = `
 Object {
-  "": Object {
-    "description": "This is an unnamed slot (the default slot)",
-    "name": "",
+  "attributes:back-side": Object {
+    "description": "Indicates that the back of the card is shown",
+    "name": "back-side",
     "required": false,
     "table": Object {
-      "category": "slots",
+      "category": "attributes",
       "defaultValue": Object {
-        "summary": undefined,
+        "summary": "false",
       },
       "type": Object {
-        "summary": undefined,
+        "summary": "boolean",
+      },
+    },
+    "type": Object {
+      "name": "void",
+    },
+  },
+  "attributes:header": Object {
+    "description": "Header message",
+    "name": "header",
+    "required": false,
+    "table": Object {
+      "category": "attributes",
+      "defaultValue": Object {
+        "summary": "\\"Your Message\\"",
+      },
+      "type": Object {
+        "summary": "string",
+      },
+    },
+    "type": Object {
+      "name": "void",
+    },
+  },
+  "attributes:rows": Object {
```

Comment:
these have appeared because we're no longer overwriting `rows` the attribute with `rows` the property. however, the input element doesn't specify `rows` to be reflected as an attribute... so this shouldn't happen. either WCA is producing some funky custom-elements.json results or our input element is somehow wrong